### PR TITLE
setup infra

### PR DESCRIPTION
**Pull request by kudaMhizha**

### CODE REVIEW

File: infra/stack/rds.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+export function createDBInstance() {
+  const dbSecret = new aws.kms.Key("example", { description: "Example KMS Key" });
+
+  const db = new aws.rds.Cluster("postgresql", {
+    availabilityZones: [
+      "eu-west-1a",
+      "eu-west-1b",
+      "eu-west-1c",
+    ],
+    backupRetentionPeriod: 5,
```
Comment (ghost) [inline infra/stack/rds.ts:13]: @kudaMhizha going with Aurora here ?

**Review by ghost — APPROVED** [review]
👍

File: infra/stack/rds.ts (hunk 1 above)
Comment (kudaMhizha) [reply infra/stack/rds.ts:13]: @kudzai-sovtech nah was just testing it out, what do you think?

Comment (ghost) [reply infra/stack/rds.ts:13]: @kudaMhizha just thinking about the cost, but we're just testing.

**Pull request merged**